physx/include/foundation/PxPreprocessor.h

```diff
@@ -99,10 +99,15 @@ Operating system defines, see http://sourceforge.net/p/predef/wiki/OperatingSyst
 	#define PX_ANDROID 1
 #elif defined(__linux__) || defined (__EMSCRIPTEN__) // note: __ANDROID__ implies __linux__
 	#define PX_LINUX 1
-#elif defined(__APPLE__) && (defined(__arm__) || defined(__arm64__))
-	#define PX_IOS 1
 #elif defined(__APPLE__)
-	#define PX_OSX 1
+	#include <TargetConditionals.h>
+	#if TARGET_OS_IPHONE
```

Comment:
TARGET_OS_IPHONE is true for any device that runs an iOS fork (watch, ipad, homepod, bridgeos, & catalyst apps) is that what you mean hear or should this be checking for iOS specifically?

Reply:
My read from the original commit is that the IOS check was predicated on the arm vs x86 detection. Luckily, I don't think we need to be worried about running physx 5 on an Apple watch 🤭

Reply:
Yeah, I went more generic here, PhysX only needs to make the distinction for IOS (iphone, ipads) and OSX (macs), PhysX SDK only compiles for those 2, so this should be enough. In the future if we add iWatch target then we can change this and make it more specific :)

Reply:
catalyst is probably the one to worry about